### ISSUE TITLE
Syntaxerror: future feature annotations is not defined

### ISSUE THREAD

**Issue by ojhall94** [KeplerGO/lightkurve#772]
I receive the following error when installing and trying to import Python3 in a Google Colab environment:

```python3
!pip install -q --upgrade git+https://github.com/KeplerGO/lightkurve.git
import lightkurve as lk

>>>  File "/usr/local/lib/python3.6/dist-packages/lightkurve/lightcurve.py", line 2
    from __future__ import annotations
                                      ^
>>> SyntaxError: future feature annotations is not defined
```

**Comment by barentsen** [comment]
Oops, Google Colab only appears to support Python 3.6.

The incompatible import is related to type hinting, which isn't very important for now, so I'll remove it.

**Comment by barentsen** [comment]
Thanks for reporting @ojhall94.  This has been resolved via #773.  Please try again!

**Comment by ojhall94** [comment]
Can confirm its fixed! Thanks @barentsen!
